tools/Generate-CodepointWidthsFromUCD.ps1

```diff
@@ -260,21 +248,29 @@ If (-not $NoOverrides) {
     }
 }
 
+$ranges.RemoveAll({ $args[0].Width -eq [CodepointWidth]::Narrow }) | Out-Null

```

Comment:
Moving this down, past the `If (-not $NoOverrides) {` block ensures that we filter out those 3 narrow codepoint overrides. Also I felt like this `RemoveAll` call would be more robust than the `Where-Object` block above.